.eslintrc.js

```diff
@@ -1,27 +1,20 @@
 module.exports = {
-    env: {
-        browser: true,
-        commonjs: true,
-        es6: true,
-        'jest/globals': true
-    },
-    extends: [
-        'standard',
-        'plugin:vue/recommended'
-    ],
-    plugins: [
-        'jest'
-    ],
-    parserOptions: {
-        ecmaVersion: 8,
-        sourceType: "module"
-    },
-    globals: {
-        Tether: true,
-        Promise: true,
-        Vue: true
+  extends: ['standard', 'plugin:vue/recommended'],
+  plugins: ['jest'],
+  parserOptions: {
+    ecmaVersion: 8,
+    sourceType: 'module'
+  },
+  env: {
+    browser: true,
+    es6: true,
+    'jest/globals': true
+  },
+  globals: {
+    Tether: true,
```

Comment:
Actually we can remove this line now.. since we no longer user `Tether` for tooltip positioning.